regulations/views/reader.py

```diff
@@ -59,3 +60,61 @@ def get_neighboring_sections(self, label, version):
             p_sect, n_sect = nav_sections
             return {'previous': p_sect, 'next': n_sect,
                     'page_type': 'reg-section'}
+
+
+class PartReaderView(ReaderView):
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        part = context['part']
+        version = context['version']
+        first_section = utils.first_section(part, version)
+        first_subpart = utils.first_subpart(part, version)
+
+        c = {
+            'subpart_view_link': reverse('subpart_reader_view', args=(part, first_subpart, version)),
+            'section_view_link': reverse('section_reader_view', args=(part, first_section, version)),
+        }
```

Comment:
move this into a function with a common name across the classes.

Reply:
Implemented in c8537d2